pkg/asset/machines/vsphere/machinesets.go

```diff
@@ -74,7 +65,136 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 			},
 		},
 	}
-	machinesets = append(machinesets, mset)
+	return mset, nil
+}
+
+func getVCenterFromServerName(server string, platformSpec *vsphere.Platform) (*vsphere.VCenterSpec, error) {
+	for _, vCenter := range platformSpec.VCenters {
+		if vCenter.Server == server {
+			return &vCenter, nil
+		}
+	}
+	return nil, errors.Errorf("%s is not a defined vCenter server", server)
```

Comment:
Do we ever expect this to fail? Should this be part of the validation?

Reply:
yep, it is [checked](https://github.com/openshift/installer/pull/5911/commits/e81b2abbe82a163823d7920b78bbae581f1caba8#diff-2cdb5d09122e1d26502732ec3ee5fb818c7153af73b55c95cc5f41c9ae005febR217-R220) during validation as well.  i'll strike the error from here.